### ISSUE TITLE
How to load file using io.open('somefile', 'rb')

### ISSUE THREAD

**Issue by abcnet** [clementfarabet/torch-ios#15]
Hi,
I have a question about how to load a file on iOS torch. I tried every effort (lfs, paths.files, ...) to get current working path of the lua file so that I could add files to that directory, but failed. Can you please help me out? Thanks a lot!
By the way, it seems some packages do not work on iOS torch, such as nngraph, xlua, and gnuplot.

**Comment by soumith** [comment]
Could you see the example, maybe you can get hints there: https://github.com/clementfarabet/torch-ios/tree/ios/ios_examples/XOR_Example

**Comment by abcnet** [comment]
Yes I did. But this example only referenced one .lua file, and no other external files were loaded in the .lua file. So I am wondering if we can import an image file and process it inside the .lua file using io.open('image', 'rb'). But the problem is the image file could not be accessed even put in the same directory as the .lua file.

**Comment by kurtjacobsdev** [comment]
Hi @abcnet I will upload an example of how to load a pre-trained network model (ill call it XOR_Example_Pre) you should be able to leverage the same code to load up any file for use inside of Lua. I will try to do so in the next few days and submit a pull request from my branch.

**Comment by abcnet** [comment]
Hi @woffle. Does the latest commit already include the new example or you will upload it later?
Thanks!

**Comment by kurtjacobsdev** [comment]
@abcnet I'll have time some free time over this weekend to add the code and then ill issue a pull request. hope thats okay.

**Comment by abcnet** [comment]
@woffle Great. Thanks !

**Comment by kurtjacobsdev** [comment]
@abcnet Hi just committed the updates to load files inside lua scripts. You can checkout my branch or wait for @soumith to merge into the main branch. Hope it works out for you.

**Comment by abcnet** [comment]
Great thanks!
